requirements.txt

```diff
@@ -1,3 +1,4 @@
+git+https://github.com/City-of-Helsinki/django-helusers.git#egg=django-helusers
```

Comment:
This will always get the `master` version of helusers, with no pinning whatsoever. Is that a good idea?